src/index.tsx

```diff
@@ -3,6 +3,8 @@
  * Modify `scripts/generate-mjml-react.ts` to make changes to these files
  */
 
+export * from "./utils";
```

Comment:
I think it prioritizes the file first, because I noticed to import from `/dist/utils/index.js`, but I agree that it is confusing. I will try the suggestion you made, as that may actually allow us to avoid it being a breaking change